.github/workflows/tm.integration.tests.yml

```diff
@@ -52,3 +52,10 @@ jobs:
         uses: ./.github/actions/tm-integration-tests
         env:
           GOPATH: /github/workspace
+      - name: go-version
```

Comment:
`go-version` needs to be the step `id`, not the step `name` in order for the next step to be able to reference this step as `steps.go-version`.